Expensify/App
# Block Virtual keyboard on Request Money Page

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7023

### Tests | QA Steps
  1. Go to staging.new.expensify.com on Mobile Web.
  2. Log in with any account
  3. Tap on Green Plus button
  4. Tap on Split/Request money.
  5. Tap on Amount field. 
  6. No virtual keyboard should pop up.

### Tested On


### Screenshots


#### Web | Desktop
![image](https://user-images.githubusercontent.com/24370807/148937058-9b706b1a-2c05-4843-b35d-78af0d570e20.png)

#### Mobile Web

![image](https://user-images.githubusercontent.com/24370807/148937112-3eee4bc4-406f-4e69-a919-351127559258.png)

#### iOS


#### Android

![image](https://user-images.githubusercontent.com/24370807/148941618-584f28b4-0280-4a51-8610-89d666984a45.png)
